orchagent/crmorch.cpp

```diff
@@ -457,10 +457,14 @@ void CrmOrch::getResAvailableCounters()
                 sai_status_t status = sai_switch_api->get_switch_attribute(gSwitchId, 1, &attr);
                 if (status != SAI_STATUS_SUCCESS)
                 {
-                    if(status == SAI_STATUS_NOT_SUPPORTED)
+                    if((status == SAI_STATUS_NOT_SUPPORTED) ||
```

Comment:
need to add status == SAI_STATUS_NOT_IMPLEMENTED

Reply:
@lguohan, Please note that Mellanox SAI is not returning SAI_STATUS_NOT_IMPLEMENTED. We can add it to avoid a possible failure or any other SAI implementation.

Reply:
yes, that is the right.